### PR TITLE
python3Packages.unstructured: 0.16.8 -> 0.16.10

### PR DESCRIPTION
**Pull request by happysalada**
## Things done

update and restructure dependencies according to the current state of the repo

@mentalblock this should fix the structure of the dependencies for unstructured.
Let me know if you still see problems.
(I've commented out unpackaged dependencies, anyone is welcome to add them, I'll be happy to review and help merge).

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/development/python-modules/unstructured/default.nix [hunk 1]
```diff
@@ -57,39 +113,7 @@
...
-  version = "0.16.8";
-  optional-dependencies = {
-    huggingflace = [
-      langdetect
-      sacremoses
-      sentencepiece
-      torch
-      transformers
-    ];
-    local-inference = [ unstructured-inference ];
```
Comment (getchoo) [inline pkgs/development/python-modules/unstructured/default.nix:69]: This breaks eval for `unstructured-api`

Comment (happysalada) [reply pkgs/development/python-modules/unstructured/default.nix:69]: Yes i saw, i basically have to do the same for that package :-(

**Comment by drupol** [comment]
This needs to be rebased on top of master branch now that it has been updated to 0.16.11 by ofborg. I'll set it to draft in the meantime, feel free to mark it as ready when you feel it is ready to be reviewed again.

**Comment by happysalada** [comment]
I'm not sure why this was closed, I've reopened
https://github.com/NixOS/nixpkgs/pull/365635
I have done a poor job with one of the dependencies, but hopefully all should pass now.

**Pull request closed without merging**